base/src/org/adempiere/util/ProcessUtil.java

```diff
@@ -160,11 +166,6 @@ public static boolean startJavaProcess(Properties ctx, ProcessInfo pi, Trx trx,
 			return false;
 		}
 		
-		if (processClass == null) {

```

Comment:
![imagen](https://github.com/adempiere/adempiere/assets/20288327/eb8786c2-084b-4ab7-a27f-bcc8b607224b)
My editor tells me it's dead code, I can revert it if you prefer.

Reply:
@EdwinBetanc0urt Hello, you see that the code is dead because the validation of the class is done after the try, the correct thing is that the validation of the null class should be before the try, so the correct thing is not to remove the code but to move it before the try